boxoffice/templates/free_order_confirmation_mail.html.jinja2

```diff
@@ -0,0 +1,233 @@
+      color: #1f2d3d;
+      margin: 7px auto 15px;
+    }
+
+    .content-wrapper {
+      padding: 10px;
+    }
+
+    .receipt-no {
+      color: #8492a6;
+      font-size: 16px;
+      float: right;
+    }
+
+    .content {
+      clear: both;
+      color: #1f2d3d;
+      font-size: 16px;
+    }
+
+    .name {
+      margin-bottom: 10px;
+    }
+
+    .btn-wrapper {
+      text-align: center;
+      margin: 25px auto;
+    }
+
+    .button {
+      border-width: 0px 0px 3px;
+      border-style: solid;
+      border-radius: 5px;
+      padding: 6px 12px;
+      color: #fff;
+      font-size: 15px;
+      display: inline-block;
+      background-color: #5BC0DE;
+      border-color: #46B8DA;
+    }
+
+    .line-break {
+      color: #EFF2F7;
+      margin: 10px -10px;
+    }
+
+    .ticket-box-title {
+      color: #1f2d3d;
+      font-size: 32px;
+      text-align: center;
+      margin: 25px auto;
+    }
+
+    table {
+      border-collapse: collapse;
+      width: 100%;
+    }
+
+    table th {
+      border: 1px solid transparent;
+      border-bottom: 1px solid #eff2f7;
+      padding: 10px;
+    }
+    table td {
+      border: 1px solid #eff2f7;
+      padding: 10px;
+    }
+
+    .center {
+      text-align: center;
+    }
+
+    .footer {
+      background-color: #f9fafc;
+      padding: 10px;
+      color: #8492a6;
+      margin: 25px 0 0;
+    }
+
+    .footer .footer-content a {
+      font-size: 14px !important;
+      text-decoration: none !important;
+      color: #8492A6 !important;
+    }
+
+    .footer-left-content {
+      float: left;
+    }
+
+    .footer-left-content h2 {
+      margin: 5px 0 5px;
+      font-size: 14px;
+      font-weight: 500;
+    }
+
+    .footer-right-content {
+      float: right;
+    }
+    
+    .footer-title {
+      font-size: 12px;
+    }
+
+    .footer-content {
+      font-size: 14px;
+    }
+
+    .clear {
+      clear: both;
+    }
+  </style>
+</head>
+
+<body>
+  <div>
+
+    <div class="heading">
+      {%- if org.details.get('logo') %}
+        <div class="logo-wrapper">
+         <img src="{{ org.details.get('logo') }}" class="logo"/>
+        </div>
+      {%- endif %}
+      <h1 class="ic-title">{{ order.item_collection.title }}
+    </div>
+
+    <div class="content-wrapper">
+
+      <p class="receipt-no">Order no: {{ order.invoice_no }}</p>
+
+      <p class="content name">Hello {{ order.buyer_fullname }},</p>
+      <p class="content">Please find your ticket details below. Looking forward to seeing you at the event!</p>
+      <div class="btn-wrapper"><a class="button" href={{"{base_url}/order/{access_token}/ticket".format(base_url=base_url, access_token=order.access_token)}} target="_blank">Fill attendee details</a></div>
+
+      <hr class="line-break">
+
+      <p class="ticket-box-title">Here {% if line_items|length > 1 %} are your <b>tickets</b>{% else %} is your <b>ticket</b>{% endif %}!</p>
+      <table>
+        <tr>
+          <th>Ticket No.</th>
```

Comment:
`Ticket no.`, since we're using sentence case.

Reply:
Fixed